core/Stream/hsStream.h

```diff
@@ -87,7 +87,11 @@ class HSPLASMA_EXPORT hsStream
 
 class HSPLASMA_EXPORT hsFileStream : public hsStream {
 protected:
+#ifdef WIN32
+    void* F;
+#else
```

Comment:
```suggestion
#ifdef _WIN32
    /* HANDLE */ void* F;
#else
```
Just so it's clear what we're actually doing here...  >.>